datastore-v1-proto-client/src/main/java/com/google/datastore/v1/client/RemoteRpc.java

```diff
@@ -46,6 +47,8 @@ class RemoteRpc {
   @VisibleForTesting static final String API_FORMAT_VERSION_HEADER = "X-Goog-Api-Format-Version";
   private static final String API_FORMAT_VERSION = "2";
 
+  @VisibleForTesting static final String X_GOOG_REQUEST_PARAMS_HEADER = "x-goog-request-params";
```

Comment:
This makes me think that we should expose this key as public in gax-java in the future, we have the same key defined [here](https://github.com/googleapis/gax-java/blob/a1d62706624539a699745a22241c9ebc14b3495d/gax-grpc/src/main/java/com/google/api/gax/grpc/CallOptionsUtil.java#L48)

Reply:
@blakeli0 makes sense! This module doesn't depend on gax, but let me know if you'd like any similar changes here.